News/Controllers/IdentitiesController.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using News.Entities.Models;
+using News.Services;
+
+namespace News.Controllers;
+
+[ApiController]
+[Route("identities")]
```

Comment:
u need use slug name here, like [Route("test-tokens")]

Reply:
this is **identity** not **id** and **entity**

Reply:
[Route("identities")] ok